### PR TITLE
Make error handling consistent across components

### PR DESCRIPTION
**Pull request by bill-filler**
Ensure error gets cleared when fetching begins on all reducers
Fix error handling on JournalPage and create reusable components for displaying errors (thanks Enterprise team!)

screenshot from 2018-09-25 15-17-50
screenshot from 2018-09-25 15-18-13

### CODE REVIEW

File: src/components/StatusAlert/index.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  return (
+    <Alert
+      className={className}
+      alertType={alertType}
+      dismissible={false}
+      dialog={
+        <div className={
+          classNames({
+            'd-flex': iconClassName && iconClassName.length > 0,
```
Comment (tuchfarber) [inline src/components/StatusAlert/index.jsx]: It should be fine to always be display: flex even if there's no icon. I would just set it in scss and remove this.

File: src/components/ErrorPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+  </div>
+);
+
+const ErrorPage = (props) => {
+  const errorMessage = props.message || 'An unknown error has occured.';
+
+  return (
+    <div className="container">
+      {props.status === 404 ? (
```
Comment (tuchfarber) [inline src/components/ErrorPage/index.jsx]: We may want to make a specific error page for 500 errors so the user knows it on our end not theirs. Fine with this being in a followup though.

Comment (bill-filler) [reply src/components/ErrorPage/index.jsx]: good suggestion, will create a story for it for future

**Comment by bill-filler** [comment]
@tuchfarber comments fixed on this

**Pull request merged**